test/unit/runner.spec.js

```diff
@@ -285,21 +294,26 @@ describe('Runner', function () {
         2
       ];
       runner.on('fail', function (test, err) {
-        err.message.should.equal('the array [\n  1\n  2\n] was thrown, throw an Error :)');
+        expect(err.message).to.equal('the array [\n  1\n  2\n] was thrown, throw an Error :)');
         done();
       });
       runner.fail(test, err);
     });
 
     it('should recover if the error stack is not writable', function (done) {
+      if (!Object.create) {
```

Comment:
checking for `Object.defineProperty` seems reasonable, but it isn't really, because IE8 implements the function incorrectly.